p2p/p2pimpl/p2pimpl.go

```diff
@@ -1,25 +1,57 @@
 package p2pimpl
 
 import (
+	"fmt"
+	"net"
+
 	"github.com/harmony-one/harmony/p2p"
 	"github.com/harmony-one/harmony/p2p/host"
 	"github.com/harmony-one/harmony/p2p/host/hostv1"
 	"github.com/harmony-one/harmony/p2p/host/hostv2"
+
+	"github.com/harmony-one/harmony/internal/utils"
+	peer "github.com/libp2p/go-libp2p-peer"
+	multiaddr "github.com/multiformats/go-multiaddr"
```

Comment:
[nit/style] This package is `ma` by default, and people's eyes have been trained to expect `ma` to mean the multiaddr package.  Do we have a strong reason to use an alternative package tag?